tests/tests_test.go

```diff
@@ -0,0 +1,278 @@
+			if err != nil {
+				log.Println(err)
+			}
+			storage, err := resource.ParseQuantity("2Gi")
+			if err != nil {
+				log.Println(err)
+			}
+			pvc := &v1.PersistentVolumeClaim{
+
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "long-name-123456789011121314151617181920212223242526272829303132",
+					Namespace: "ocp-41583-longpvcname",
+				},
+				Spec: v1.PersistentVolumeClaimSpec{
+					AccessModes: []v1.PersistentVolumeAccessMode{"ReadWriteOnce"},
+					Resources: v1.ResourceRequirements{
+						Requests: v1.ResourceList{
+							"storage": storage,
+						},
+					},
+				},
+			}
+			_, err = sourceClient.CoreV1().PersistentVolumeClaims("ocp-41583-longpvcname").Create(ctx, pvc, metav1.CreateOptions{})
+			if err != nil {
+				log.Println(err)
+			}
+			res := int32(1)
+			deployment := &appsv1.Deployment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "longpvc-test",
+					Namespace: "ocp-41583-longpvcname",
+					Labels: map[string]string{
+						"app": "longpvc-test",
+					},
+				},
+				Spec: appsv1.DeploymentSpec{
+					Replicas: &res,
+					Selector: &metav1.LabelSelector{
+						MatchLabels: map[string]string{
+							"app": "longpvc-test",
+						},
+					},
+					Template: v1.PodTemplateSpec{
+						ObjectMeta: metav1.ObjectMeta{
+							Labels: map[string]string{
+								"app": "longpvc-test",
+							},
+						},
+						Spec: v1.PodSpec{
+							Containers: []v1.Container{
+								v1.Container{
+									Name:            "pod-test",
+									ImagePullPolicy: "Always",
+									Image:           "quay.io/openshifttest/alpine",
+									Command:         []string{"/bin/sh", "-c", "--"},
+									Args:            []string{"while true; do sleep 30; done;"},
+									VolumeMounts: []v1.VolumeMount{
+										v1.VolumeMount{
+											Name:      "testvolume",
+											MountPath: "/data/test",
+										},
+									},
+								},
+							},
+							Volumes: []v1.Volume{
+								v1.Volume{
+									Name: "testvolume",
+									VolumeSource: v1.VolumeSource{
+										PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
+											ClaimName: "long-name-123456789011121314151617181920212223242526272829303132",
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			}
+			_, err = sourceClient.AppsV1().Deployments("ocp-41583-longpvcname").Create(ctx, deployment, metav1.CreateOptions{})
+			if err != nil {
+				log.Println(err)
+			}
+			if err != nil {
+				log.Println(err)
+			}
+			Eventually(func() string {
+				podList, err := sourceClient.CoreV1().Pods("ocp-41583-longpvcname").List(ctx, metav1.ListOptions{})
+				if err != nil {
+					log.Println(err)
+				}
+				for _, p := range podList.Items {
+					if p.Status.Phase == "Running" {
+						podCmd := pods.PodCommand{
+							Pod:     &p,
+							RestCfg: sourceCfg,
+							Args:    []string{"sh", "-c", "dd if=/dev/zero of=/data/test/file_1 bs=60M count=1"},
+						}
+						err = podCmd.Run()
```

Comment:
I am failing to understand what is the point of running this command here, 

1. the output is not saved,
2. the output does not determine the result of this Eventually block condition

What would happen if we removed this?

Reply:
actually never mind, you are generating the data, with this, I misread!!